canopy/canopyjs/src/index.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/prop-types */
+/**
+ * Copyright 2018-2020 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, createContext, useContext, useEffect } from 'react';
+import { get } from './request';
+
+import {
+  mountCurrentSapling,
+  mountSaplingStyles,
+  mountConfigSaplings
+} from './loadSaplings';
+
+export const CanopyContext = createContext({});
+
+const fetchUserSaplings = async saplingURL => {
+  const response = await get(`${saplingURL}/userSaplings`);
+  const userSaplingsResponse = response.json;
+  return userSaplingsResponse;
+};
+
+const fetchConfigSaplings = async saplingURL => {
+  const response = await get(`${saplingURL}/configSaplings`);
+  const configSaplingsResponse = response.json;
+  return configSaplingsResponse;
+};
+
+export function CanopyProvider({ saplingURL, splinterURL, children }) {
```

Comment:
Could this and its associated methods get moved into a canopy-provider.js file?

Reply:
Is there a specific rationale for that? CanopyProvider and the useContext wrappers are the only things that canopyJS is going to export, so I didn't think there was much of a point to move them to a separate file.

Reply:
After an in person discussion, we decided to resolve this in a future PR.